profiles/base.yaml

```diff
@@ -721,10 +721,21 @@ mining_implement: shovel
 mine_use_packet: true
 mine_while_training: false
 
+## Locksmithing Settings ##
+#https://elanthipedia.play.net/Lich_script_repository#locksmithing
+# use ;pick and pick settings first
+pick_live_boxes: true
+# second option to a daily use lockbox trainer like a training box or harvest bag
+have_training_box: false
+picking_lockbox: training box
+picking_worn_lockbox: false
+# consumable boxes will be pulled from your burgle loot_container and should be kept separate from live boxes.
+# list as many as you want - though it will try from the top down.
+consumable_lockboxes:
```

Comment:
❌ This should have an entry in `base-empty.yaml`

Reply:
added.